src/AmbientSounds.Uwp/Services/MediaPlayerService.cs

```diff
@@ -89,7 +91,22 @@ public void Play(Sound s)
             else
             {
                 _player.Pause();
-                _player.Source = MediaSource.CreateFromUri(new Uri(s.FilePath));
+
+                var mediaSource = MediaSource.CreateFromUri(new Uri(s.FilePath));
+                _player.Source = mediaSource;
+
+                // Set data in System Media Transport Controls
+                // This step must be done after setting the player source
+                var playbackItem = MediaPlaybackItem.FindFromMediaSource(mediaSource);
+                var props = playbackItem.GetDisplayProperties();
+                props.Type = MediaPlaybackType.Music;
+                props.MusicProperties.Title = s.Id;
```

Comment:
Can we `= s.Name ?? s.Id`

In the future, Name will be used for the translated name of the sound.

Reply:
Updated. Works as expected.